Quandl.Excel.Console/SetupHelp.cs

```diff
@@ -11,17 +14,49 @@ public class SetupHelp
         private const string AddinPartialPackageString = "Quandl";
         private const string AddinPackageString = "Quandl.Excel.UDF.Functions-AddIn.xll";
 
-        private static readonly string OpenValue = $@"/R ""{Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData)}\Quandl\Quandl.Excel.UDF.Functions-AddIn.xll""";
+        private static string Excel32InstallFolder = $@"/R ""C:\Program Files(x86)\Microsoft Office\Office{0}\EXCEL.EXE""";
+    
+        private static string _openValue = null;
+        private static bool _is64Bit = Is64BitOperatingSystem();
+        private static string _version = null;
```

Comment:
I think this needs to be more descriptive. Is this the excel version or the windows version?